dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+
+<a name="native-icon" />
+
+##### `adUnit.mediaTypes.native.icon`
+
+{: .table .table-bordered .table-striped }
+| Name            | Scope    | Type                                  | Description                                                                                                                       |
+|-----------------+----------+---------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------|
+| `required`      | Requird  | Boolean                               | Whether an icon asset is required on this ad.                                                                                     |
+| `sizes`         | Optional | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                           |
+| `aspect_ratios` | Optional | Array[Object]                         | Instead of `sizes`, you can define allowed aspect ratios.  For properties, see [`icon.aspect_ratios`](#native-icon-aspectratios). |
+
+<a name="native-icon-aspectratios" />
+
+###### `adUnit.mediaTypes.native.icon.aspect_ratios`
+
+{: .table .table-bordered .table-striped }
+| Name           | Scope | Type | Description |
+|----------------+-------+------+-------------|
+| `min_width`    | Optional | Integer | The minimum width required for an image to serve (in pixels).                                        |
+| `ratio_height` | Required | Integer | This, combined with `ratio_width`, determines the required aspect ratio for an image that can serve. |
+| `ratio_width`  | Required | Integer | See above.                                                                                           |
+
+<a name="video" />
+
+#### `adUnit.mediaTypes.video`
+
+{: .table .table-bordered .table-striped }
+| Name             | Scope    | Type                   | Description                                                                                                                                             |
+|------------------+----------+------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `context`        | Required | String                 | The video context, either `"instream"` or `"outstream"`.  Example: `context: "outstream"`                                                               |
+| `playerSize`     | Optional | Array[Integer,Integer] | The size (width, height) of the video player on the page, in pixels.  Example: `playerSize: [640, 480]`                                                 |
+| `mimes`          | Optional | Array[String]          | Required by OpenRTB when using [Prebid Server]({{site.baseurl}}/dev-docs/get-started-with-prebid-server.html).  Example: `mimes: ['video/mp4']`         |
```

Comment:
The note about "Required by Prebid Server" should feature more prominently.  Let's change the scope to something like "Recommended.  Required by OpenRTB when using Prebid Server...". 

Description should be taken from OpenRTB spec:

Content MIME types supported (e.g., “video/x-ms-wmv”,
“video/mp4”)

Reply:
Updated in 53a6e3f